Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -130,11 +130,16 @@ function xASL_wrp_ResampleASL(x)
 %% ------------------------------------------------------------------------------------------
 % 6. Create mean control image, if available, in native & standard space
 if  nVolumes>1    
-    % Create mean control in native space
-    ControlIm = xASL_quant_GetControlLabelOrder(xASL_io_Nifti2Im(x.P.Path_rdespiked_ASL4D));
-    IM_mean = xASL_stat_MeanNan(ControlIm, 4);
-    xASL_io_SaveNifti(x.P.Path_rdespiked_ASL4D, x.P.Path_mean_control, IM_mean, [], 0);
-
+    if x.modules.asl.bTimeEncoded % ===== Here we should just configure the mean control (first volume)==== %
+        Im=xASL_io_Nifti2Im(x.P.Path_rdespiked_ASL4D);   

```

Comment:
Space missing (`Im = xASL_io_Nifti2Im`). And good to use `Image` rather than `Im`, make the code easy to read and understand!

Reply:
@HenkMutsaerts... this is critique on an extremely high level. This shouldn't be a change request and rather a small comment. I bet I can find old code that you wrote without those spaces 😉